changelog/fix_redundant_presence_validation_with_multi_attributes.md

```diff
@@ -0,0 +1 @@
+* [#609](https://github.com/rubocop/rubocop-rails/pull/609): Fix autocorrection of multiple attributes for `Rails/RedundantPresenceValidationOnBelongsTo` cop. ([@pirj][])
```

Comment:
Can you specify the original issue address instead?

```suggestion
* [#603](https://github.com/rubocop/rubocop-rails/issues/603): Fix autocorrection of multiple attributes for `Rails/RedundantPresenceValidationOnBelongsTo` cop. ([@pirj][])
```

Reply:
Done 👍